.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches:
+        - main
+        - feature/SCKT-1_init_project_supaapps_api_kit_client
```

Comment:
all branches should build not main & this branch so we can always test PR

Reply:
Done ✅